frontend/public/background.css

```diff
@@ -4,7 +4,7 @@
     width: 100vw;
     color: var(--forever-susan-accent-yellow);
     background-color: var(--forever-susan-dark-crimson);
-    background-image: url("background_image.jpeg");
+    background-image: url('background_image.jpeg');
```

Comment:
this change isn't needed since the issue it was attempting to address was from unresolved merge conflicts in the file which was fixed in https://github.com/JamesKoenig/FieldLily/pull/131/commits/f0eca000a4d92c44342fc8642a72cfd1491ded2d